### PR TITLE
Release openmmtools 0.17.0

### PR DESCRIPTION
**Pull request by andrrizzi**

### CODE REVIEW

**Comment by andrrizzi** [comment]
In 0.17 we dropped support for python 2.7, and I see that the [tests are failing now](https://travis-ci.org/omnia-md/conda-recipes/jobs/484098787#L4049-L4050). Is this a problem? Or does it just mean that if I merge now the py2 build won't be uploaded to anaconda?

**Comment by jchodera** [comment]
Hm, it's still scheduling `openmmtools` builds for python 2.7:
```
Scheduling package: openmmtools
  full_name: openmmtools-0.17.0-py27_0.tar.bz2
  python: 27
  numpy: 1.14
```

File: openmmtools/meta.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  version: 0.17.0
 
 source:
-    fn: 0.16.0.tar.gz
-    url: https://github.com/choderalab/openmmtools/archive/0.16.0.tar.gz
+    fn: 0.17.0.tar.gz
+    url: https://github.com/choderalab/openmmtools/archive/0.17.0.tar.gz
 
 build:
   number: 0
```
Comment (jchodera) [inline openmmtools/meta.yaml]: You want to skip `py2k` below if we have dropped Python 2.7 support. 
I'd change line 11 from
```yaml
  skip: True  # [win32 or (win and py2k)]
```
to
```yaml
  skip: True  # [py2k or win32 or (win and py2k)]
```

Comment (andrrizzi) [reply openmmtools/meta.yaml]: Thanks!

**Comment by jchodera** [comment]
Builds have been pushed!
https://anaconda.org/omnia/openmmtools/files

**Pull request merged**